README.md

```diff
@@ -7,6 +7,8 @@ Currently, a Google version is in place until such time as this app is launched.
 
 This is drawn from the [SoSAGleanTN repo](https://github.com/SoSAGleanTNorg/GleanTnWeb).
 
+
+The currently deployed firebase app built with jquery is available here: [Donor Connection Firebase App](https://gleantn-1794b.firebaseapp.com)
```

Comment:
This is going into the React branch. When this is merged we won't need to point to the jquery app anymore.